README.md

```diff
@@ -1,2 +1,25 @@
-# denuncia-discriminacao
-Em princípio, um apanhado de ideias vivas sobre uma possível plataforma de denúncia de diversos tipos de discriminação, começando por racismo.
+# Denúncia Discriminação
+
+## Ideia
+Uma plataforma de denúncia de diversos tipos de discriminação, tendo racismo como o primeiro MVP.
+
+## Benefícios sociais
+- agregar e organizar reclamações, tendo assim efeito potencializador
+- notificar as pessoas que denunciaram um determinado estabelecimento ou marca, quando outras denúncias relacionadas vierem a ocorrer, gerando encorajamento e senso de pertencimento
+- permitir, de maneira trivial, o compartilhamento em redes sociais, não só das ocorrências associadas a um estabelecimento ou marca específica, como de gráficos e métricas refletindo diversas perspectivas
+- mecanismo que, mediante aceitação des denunciantes, permitiria agrupar pessoas lesadas por um mesmo estabelecimento ou marca visando a possibilidade de ações coletivas de responsabilização/reparação
+- viabilizar, através da ação da imprensa e de entidades antirracistas, a pressão pública sobre os estabelecimentos ou marcas mais ofensoras, o que poderia gerar uma reação positiva por parte das acusadas ou, em caso de leniência, prejuízos legais e/ou desfinanciamentos
+- incentivar/facilitar denúncias, evitando assim com que parte significativa dos eventos se oculte nas consciências indiviuais das pessoas discriminadas. O incentivo vem na medida em que as pessoas enxerguem nessa ferramenta um canal efetivo, capaz de gerar consequências tempestivas e efetivas para os estabelecimentos ou marcas ofensoras
+
+## Possíveis extensões/evoluções:
+- público lgbtqia+
+- mulheres, vítimas de assédio sexual e misoginia
+- permitir denúncias, com um nível de privacidade mais arrojado, das discriminações já contempladas pela plataforma **ocorridas em ambientes de trabalho** (de maneira a angariar a perspectiva das empresas mais ofensoras no âmbito das relações trabalhistas, e fornecer os mesmos mecanismos de coação)
+
+## Questões essenciais em aberto:
+- como financiar?
+- é juridicamente viável?
```

Comment:
Thiago: Essa iniciativa tem uma série de implicações legais
Thiago: Vc está partindo da premissa de que todas as pessoas são honestas
Thiago: Mas uma imputação falsa de crime a alguém tbem e crime
Thiago: E vc estará legitimando essa situação 
Poderá ser responsabilizado civilmente e ter que pagar indenização pesada
Thiago: Mesmo assim se quiser correr o risco 
A plataforma teria que ser uma espécie de reclame aqui, vc teria que dar o direito de resposta a outra parte interessada
Thiago: Se virar febre saiba que o judiciário exigirá informações a respeito de quem realizou a denuncia anônima, então em que pese constar como anônima para outros usuários o interessante seria vc fazer a parte criar um cadastro com alguma.forma de confirmação da identidade, tipo foto pelo próprio aplicativo
Thiago: Pois as pessoas podem se utilizar disso para interesses escusos
Thiago: Já vimos alguns casos em que a mãe faz denuncia de estupro do pai para poder ficar com a guarda dos filhos e depois na instrução do processo fica claro que não era verdade e que a mãe inventou a história
Fernando: excelente!! Pensarei melhor sobre essas coisas todas =)
Fernando: seria pique isso mesmo... inclusive, depois de amadurecer um pouco a ideia, tá nos planos procurar o reclame aqui pra ver se lhes interessa
Fernando: obrigado ❤️
Thiago: Por nada
Thiago: Se for levar realmente a ideia a sério me avisa porque analisamos se tem alguma forma de mitigar eventual prejuízo
Fernando: boa, falo sim. A sério eu já estou levando... em breve te chamo pra bater um papo então, quando já tiver evoluído um pouco mais com outros pilares